docs/features/features-cli.mdx

```diff
@@ -3,143 +3,195 @@ id: cli
 title: Command Line Interface (CLI)
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-KintoHub has a command line interface (CLI) which can be used to debug and interact with your services.
+KintoHub provides a command line interface (CLI) which can be used to debug and interact with your services.
 
 The CLI is currently an open source **ALPHA** utility that we are working on.
-Contributions are welcomed at [kintohub/kinto-cli](https://github.com/kintohub/kinto-cli)
+Contributions are welcomed at [kintoproj/kinto-cli](https://github.com/kintoproj/kinto-cli)
 
-## Install
+## <u>Installation</u>
 
 <Tabs
   defaultValue="mac"
   values={[
     {label: 'Mac', value: 'mac'},
-    {label: 'Windows', value: 'windows'},
     {label: 'Linux', value: 'linux'},
+    {label: 'Windows', value: 'windows'}
   ]}>
   <TabItem value="mac">
     You can install kinto-cli with the following command within your terminal:
 
-    curl -L https://cli.kinto.io/install.sh | bash
-  </TabItem>
-  <TabItem value="windows">
-    You can install kinto-cli with the following command within your Powershell terminal:
+    curl -L https://raw.githubusercontent.com/kintoproj/kinto-cli/main/install.sh | bash
 
-    iex (new-object net.webclient).downloadstring('https://cli.kinto.io/install.ps1')
   </TabItem>
   <TabItem value="linux">
     You can install kinto-cli with the following command within your terminal:
 
-    curl -L https://cli.kinto.io/install.sh | bash
+    curl -L https://raw.githubusercontent.com/kintoproj/kinto-cli/main/install.sh | bash
+
+  </TabItem>
+  <TabItem value="windows">
+    You can install kinto-cli with the following command within your Powershell terminal:
+
+    Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/kintoproj/kinto-cli/main/install.ps1')
+
   </TabItem>
+ 
 </Tabs>
 
-## Upgrade
+## <u>Upgrade</u>
 
 To upgrade the CLI to the latest version, re-run the commands in the installation step.
 
-## Commands
+## <u>Commands</u>
 
-### kinto login
+## Init
 
-This will allow you to login to your KintoHub account
+```
+kinto init {coreHost}
+```
 
-### kinto env
+Switches the `coreHost` of **KintoHub** instance for future CLI commands.
```

Comment:
comment - should we explain what is the `coreHost`? that it is the URL of kinto-core? and that it needs to be external (ingress) in order to be accessed by a third party?